csrc/utils.h

```diff
@@ -29,6 +28,44 @@ struct SyclTypeTrait<c10::BFloat16> {
   using Type = sycl::ext::oneapi::bfloat16;
 };
 
+template <typename T>
+struct AccumulateType {
+ private:
+  static constexpr bool is_half = std::is_same_v<T, at::Half> ||
```

Comment:
why named `is_half` here? I understand it's copied from ipex.

Reply:
yes. half can't reflect to bf16 and fp8 . may be we can rename to is_lowp?

Reply:
is_narrow_float